airflow/api_connexion/openapi/v1.yaml

```diff
@@ -4313,13 +4313,14 @@ components:
       required: true
       description: The import error ID.
 
-    DatasetID:
+    DatasetURI:
       in: path
-      name: id
+      name: uri
       schema:
-        type: integer
+        type: string
+        format: path
```

Comment:
This was the key piece to get strings with slashes in them to be accepted. We probably need to add this to a few other parts of the API